chart/epinio/questions.yml

```diff
@@ -173,3 +173,9 @@ questions:
   required: false
   type: string
   group: "Debugging"
+- variable: rancher.url
```

Comment:
Would be possible to replace the existing one `server.accessControlAllowOrigin` by this new? 
```
server:
  # Domain which serves the Rancher UI (to access the API)
  accessControlAllowOrigin: ""
```

Reply:
Done. `server.accessControlAllowOrigin` has been replaced with `server.url`.

@enrichman Can we add something that might stand out in release notes, to ensure people who upgrade can switch over?